sw/device/lib/testing/mask_rom_test.h

```diff
@@ -0,0 +1,93 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_MASK_ROM_TEST_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_MASK_ROM_TEST_H_
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace mask_rom_test {
```

Comment:
I feel like GlobalMock is a sufficiently generic type (i.e., I want to use it for CSR mocking) that it probably doesn't want to be closely associated with the Mask ROM name.

Reply:
I agree, we can move it out when we use it somewhere else.